CONTRIBUTING.md

```diff
@@ -18,28 +20,26 @@ No software is bug free. So, if you got an issue, follow these steps:
 
 ### Testing against `master`
 
-To test your project against the current latest version of storybook, you can clone the repository and link it with `npm`. Try following these steps:
+To test your project against the current latest version of storybook, you can clone the repository and link it with `yarn`. Try following these steps:
 
 1.  Download the latest version of this project, and build it:
 
-```sh
-git clone https://github.com/storybooks/storybook.git
-cd storybook
-npm install
-npm run bootstrap -- --core
-```
+        ```sh
+        git clone https://github.com/storybooks/storybook.git
+        cd storybook
+        yarn install
+        yarn run bootstrap -- --core
```

Comment:
this should probably be updated to `yarn run bootstrap:core`

Reply:
Probably in another PR?

Reply:
By the way, it's only by fortune that those are the same things in case of `core`
For example, `yarn run bootstrap -- --react-native-vanilla` is not the same as `yarn run bootstrap:react-native-vanilla`. Looks like only the `bootstrap` command should be called immediately. @ndelangen, please correct me if I'm wrong

Reply:
@danielduan The way it is, is fine. We want people to use the bootstrap script. It's superior to remembering a list of npm script and that way we can change how we order our npm script.